contents/docs/surveys/_snippets/ios-surveys-installation.mdx

```diff
@@ -0,0 +1,68 @@
+## Step one: Add PostHog to your app
+
+> 🚧 **Note:** Mobile surveys for iOS is currently in **beta**. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asurveys%3Alow%3Atrue) or one of our other [support options](/docs/support-options).
+
+> **Note:** Using surveys requires PostHog's iOS SDK version >= 3.22.0. It's recommended to always use the latest version.
+
+import IOSInstall from "../../integrate/_snippets/install-ios.mdx"
+
+<IOSInstall />
+
+
+
+## Step two: Enable surveys in your configuration
+
+> **⚠️ Caution:**
+> - Surveys are currently in beta and only accessible through the `Experimental` SPI in Swift.
+> - We recommend thorough testing in your development environment before deploying to production.
+> - Survey APIs may change frequently and introduce breaking changes between SDK releases. Consider this when planning updates.
```

Comment:
I think the note above is enough, this only scares people from trying things, it's beta so it's clear that people have to test it